features/step_definitions/todo_steps.rb

```diff
@@ -20,6 +20,9 @@
   get_todo(@last_todo_id)
 end
 
+When 'I update that todo' do
+  update_todo(@last_todo_id)
```

Comment:
We should be more specific with the field that's being updated ?

Reply:
https://cucumber.io/docs/cucumber/cucumber-expressions/

Reply:
f8b524767b8696e30e12820161eaf896c2bd37a6